doc/whats_new/v0.22.rst

```diff
@@ -623,6 +627,10 @@ These changes mostly affect library developers.
   Such classifiers need to have the `binary_only=True` estimator tag.
   :pr:`13875` by `Trevor Stephens`_.
 
+- Estimators are expected to convert input data (``X``, ``y``,
+  ``sample_weights``) to numpy ``ndarray`` and never call
+  ``__array_function__`` on the original datatype that is passed.
```

Comment:
```suggestion
  ``__array_function__`` on the original datatype that is passed. This means that,
  by default, estimators are not expected to support the array function dispatching
  mechanism of `NEP 18`_.
```

Reply:
I don't think this wording is particularly clear. Estimators are forbidden from supporting the mechanism of NEP 18 with this PR.

Reply:
I did not want to phrase it in a way that this will always be the case in the future. But phrase it as you wish as long as NEP 18 is mentioned :)